Remotion/Security/Core/Metadata/LocalizationFileNameStrategy.cs

```diff
@@ -25,7 +25,8 @@ public class LocalizationFileNameStrategy
     public string GetLocalizationFileName (string metadataFilename, CultureInfo culture)
     {
       string baseFilename = Path.GetFileNameWithoutExtension (metadataFilename);
-      string basePath = Path.GetDirectoryName (metadataFilename);
+      // TODO: Add notnull assertion
+      string basePath = Path.GetDirectoryName (metadataFilename)!;
```

Comment:
todo with possible null-bug-guard via Assert-not-null since it should check that the metadataFilename is correctly formatted as a full path. Also, this is a public method w/o arg-checks. not nice.